src/services/GroupMeService.ts

```diff
@@ -44,7 +44,7 @@ export default class GroupMeService implements MessagingService, GroupService<Gr
     });
 
     // eslint-disable-next-line @typescript-eslint/camelcase
-    return [mentionString, { loci, type: "mentions", user_ids: users.map(u => u.user_id) }];
+    return [`${msgStart}${mentionString}`, { loci, type: "mentions", user_ids: users.map(u => u.user_id) }];
```

Comment:
${msgStart}${msgStart.length > 0 ? ' ' : ''}${mentionString} ?

Reply:
yuck imo

Reply:
we can revisit later :P